app/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "app"
+version = "0.1.0"
+description = ""
+authors = [""]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+Django = "3.0.7"
+black = "20.8b1"
+celery = "4.4.7"
+coverage = "5.2.1"
+django-cors-headers = "3.5.0"
+django-filter = "2.3.0"
+django-widget-tweaks = "1.4.8"
+djangorestframework = "3.11.0"
+dockerflow = "2020.6.0"
+factory_boy = "3.0.1"
+flake8 = "3.8.3"
+gunicorn = "20.0.4"
+ipdb = "0.13.3"
+mock = "4.0.2"
+psycopg2 = "2.8.5"
+pytest = "6.0.1"
+pytest-django = "3.9.0"
+pytest-testmon = "1.0.3"
+pytest-watch = "4.2.0"
+python-decouple = "3.3"
+raven = "6.10.0"
+redis = "3.5.3"
+requests = {version = "2.24.0", extras = ["security"]}
+whitenoise = "5.2.0"
+parameterized = "0.7.4"
+pytest-cov = "2.10.0"
+django-mozilla-product-details = "0.14.1"
+django-countries = "6.1.3"
+markus = "2.2.0"
+datadog = "0.37.1"
+sqlparse = "0.3.1"
+asgiref = "3.2.10"
+uritemplate = "3.0.1"
+packaging = "20.4"
+pyparsing = "2.4.7"
+django-markdown2 = "0.3.1"
+markdown2 = "2.3.9"
+tblib = "1.7.0"
+pytest-xdist = "1.34.0"
+djangorestframework-csv = "2.1.0"
+unicodecsv = "0.14.1"
+kinto-http = "10.7.0"
+jsonschema = "3.2.0"
+mozilla-nimbus-shared = "0.0.9"
+
+[tool.poetry.dev-dependencies]
```

Comment:
Oh sweet, this'll let us move dev dependencies in here.  Maybe someday we can do that thing where we create two docker images, one for development and one for production.  Prolly can do that outside this pr though.